### PR TITLE
Refactoring Privacy Dashboard

### PR DESCRIPTION
**Pull request by Hackwar**
What the title says. Have a look at the dashboard now, then apply this PR. Since we are close to merging over, I refrained from creating an update SQL file and instead added it to the existing one. So either hand-pick the SQL and execute it by hand or re-install. 

I've mostly copied over the code from the existing stuff, so I did not test if this works correctly here. It displays "right" and seems to be working, so that is good enough for me. The rest can be fixed in the main repo...

### CODE REVIEW

File: administrator/language/en-GB/en-GB.mod_privacy_status.sys.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_PRIVACY_STATUS="Privacy Status Check"
+MOD_PRIVACY_STATUS_XML_DESCRIPTION="The Privacy Status Check Module shows information about your sites privacy status."
+MOD_PRIVACY_STATUS_LAYOUT_DEFAULT="Default"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_privacy_status.sys.ini:7]: move up one line - alpha sorting

File: administrator/modules/mod_privacy_status/mod_privacy_status.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		<filename>helper.php</filename>
+	</files>
+	<languages>
+		<language tag="en-GB">en-GB.mod_privacy_status.ini</language>
+		<language tag="en-GB">en-GB.mod_privacy_status.sys.ini</language>
+	</languages>
+	<help key="JHELP_EXTENSIONS_MODULE_MANAGER_ADMIN_PRIVACY_STATUS" />
+	<config>
+		<fields name="params">
+			<fieldset name="basic">
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/mod_privacy_status.xml]: unless something changed we dont need this empty fieldset

File: administrator/modules/mod_privacy_status/mod_privacy_status.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+			</fieldset>
+			<fieldset name="advanced">
+				<field
+					name="layout"
+					type="modulelayout"
+					label="JFIELD_ALT_LAYOUT_LABEL"
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC"
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/mod_privacy_status.xml]: delete the description

File: administrator/modules/mod_privacy_status/mod_privacy_status.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+				/>
+
+				<field
+					name="moduleclass_sfx"
+					type="textarea"
+					label="COM_MODULES_FIELD_MODULECLASS_SFX_LABEL"
+					description="COM_MODULES_FIELD_MODULECLASS_SFX_DESC"
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/mod_privacy_status.xml]: delete the description

File: administrator/modules/mod_privacy_status/mod_privacy_status.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+				 	rows="3"
+				/>
+
+				<field
+					name="cache"
+					type="list"
+					label="COM_MODULES_FIELD_CACHING_LABEL"
+					description="COM_MODULES_FIELD_CACHING_DESC"
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/mod_privacy_status.xml]: delete the description

File: administrator/modules/mod_privacy_status/mod_privacy_status.xml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+					>
+					<option value="1">JGLOBAL_USE_GLOBAL</option>
+					<option value="0">COM_MODULES_FIELD_VALUE_NOCACHING</option>
+				</field>
+
+				<field
+					name="cache_time"
+					type="number"
+					label="COM_MODULES_FIELD_CACHE_TIME_LABEL"
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/mod_privacy_status.xml]: delete the description

File: administrator/modules/mod_privacy_status/tmpl/default.php [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+				<?php endif; ?>
+			</td>
+			<td>
+				<div><?php echo Text::_('COM_PRIVACY_STATUS_CHECK_REQUEST_FORM_MENU_ITEM_PUBLISHED'); ?></div>
+				<?php if ($requestFormPublished['link'] !== '') : ?>
+					<small><a href="<?php echo $requestFormPublished['link']; ?>"><?php echo $requestFormPublished['link']; ?></a></small>
+				<?php endif; ?>
+			</td>
+		</tr>
+	<tr>
```
Comment (brianteeman) [inline administrator/modules/mod_privacy_status/tmpl/default.php]: indent the row and subsequent lines one tab

**Review by bembelimen — APPROVED** [review]

**Pull request merged**